modules/virt-about-preallocation.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-about-preallocation_{context}"]
+= About preallocation
+
+The Containerized Data Importer (CDI) can use the QEMU preallocate mode for data volumes in order to improve write performance. Preallocation mode can be used for importing and uploading operations and when creating blank data volumes.
```

Comment:
s/in order to/to

second sentence could be changed to the active voice ("You can use preallocation mode...")